### PR TITLE
Update collection form to be consistent with updated review pages

### PR DESCRIPTION
**Pull request by howard-e**
[Preview Tests](https://deploy-preview-1017--aria-at.netlify.app)

See #1009 and #1010.

This PR updates the collection form's content to align with a subset of recent changes on the review page, introduced by #997.

**Note: These changes are branched off of #1012. That PR should be merged before these changes.**

### CODE REVIEW

File: tests/resources/aria-at-test-run.mjs [hunk 1]
```diff
@@ -155,34 +163,35 @@ export function instructionDocument(resultState, hooks) {
...
         focus: resultState.currentUserAction === UserActionMap.LOAD_PAGE,
       },
       description: `${modePhrase} how ${resultState.config.at.name} behaves when performing task "${lastInstruction}"`,
       instructions: {
-        header: 'Test instructions',
+        header: 'Instructions',
         instructions: [
           [
-            `Restore default settings for ${resultState.config.at.name}. For help, read `,
+            `Configure ${resultState.config.at.name} with default settings. For help, read `,
```
Comment (alflennik) [inline tests/resources/aria-at-test-run.mjs:174]: This line doesn't match the text I see in the issue https://github.com/w3c/aria-at/issues/1010 is there documentation for this change anywhere?

**Review by alflennik — COMMENTED** [review]
Looks great! I reviewed the preview and I can verify that the collection form now matches what I'm seeing on the latest aria-at-app versions. Code looks good. I have a couple of questions but in general it's looking great.

File: tests/resources/aria-at-test-run.mjs (hunk 1 above)
Comment (howard-e) [reply tests/resources/aria-at-test-run.mjs:174]: That's text to match against any of the [`ats[].defaultConfigurationInstructionsHTML`](https://github.com/w3c/aria-at/blob/master/tests/support.json) values

> This line doesn't match the text I see in the issue https://github.com/w3c/aria-at/issues/1010

What text are you referencing there?

Comment (alflennik) [reply tests/resources/aria-at-test-run.mjs:174]: Oh I think I misread it. Actually, it looks right.

File: tests/resources/aria-at-harness.mjs [hunk 2]
```diff
@@ -550,8 +553,9 @@ function renderVirtualInstructionDocument(doc) {
    */
   function instructionHeader({ header, description }) {
     return fragment(
-      h1(id('behavior-header'), tabIndex('0'), focus(header.focus), rich(header.header)),
-      p(rich(description))
+      h1(id('behavior-header'), tabIndex('0'), focus(header.focus), rich(header.header))
+      // No longer needed; contains the old `With {AT} in {atMode} mode, describe how {AT} behaves when performing task, {task}`
+      // p(rich(description))
```
[the remark this answers is not in the document]
Comment (howard-e) [reply tests/resources/aria-at-harness.mjs]: Done in https://github.com/w3c/aria-at/pull/1017/commits/d757a465f08f2ecadcd868e5f83b92c19bbd002c

**Review by alflennik — APPROVED** [review]
Thanks for addressing that feedback!

**Review by stalgiag — APPROVED** [review]
I haven't looked at many of these files in a long time (or ever) so it took me a while to parse what was going on. The code in many of these files feels quite complex due to inline transformation of data to match specific input/output schemas. That said, I think what you have changed here does simplify some of this logic in places which is quite a feat and hopefully part of a gradually move towards simplification and separation of concerns. Most importantly, it does it what it sets out to do (as far as my testing was able to tell)

There is a small merge conflict that needs to be resolved but otherwise I believe this is safe too merge. I am going to approv and hand it back to you for resolving that conflict.

**Review by mcking65 — APPROVED** [review]
Looks great, thanks @howard-e !

**Pull request merged**